rel/files/eunit.config

```diff
@@ -0,0 +1,16 @@
+% Licensed under the Apache License, Version 2.0 (the "License"); you may not
+% use this file except in compliance with the License. You may obtain a copy of
+% the License at
+%
+%   http://www.apache.org/licenses/LICENSE-2.0
+%
+% Unless required by applicable law or agreed to in writing, software
+% distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+% WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+% License for the specific language governing permissions and limitations under
+% the License.
+
+[
+    {kernel, [{error_logger, silent}]},
+    {sasl, [{sasl_error_logger, false}]}
```

Comment:
We probably want to add `{couch_log, [{backend, couch_log_lager}]}` here as well, though this does not reduce noise.

What do you think about adding to couch_log eunit specific 'quiet' log backed that just drops messages and adding it here?